src/test/java/org/kohsuke/github/OrgInstallationCredentialProviderTest.java

```diff
@@ -0,0 +1,32 @@
+package org.kohsuke.github;
+
+import net.sf.ezmorph.test.ArrayAssertions;
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class OrgInstallationCredentialProviderTest extends AbstractGitHubWireMockTest {
+
+    @Test(expected = HttpException.class)
+    public void invalidJWTTokenRaisesException() throws IOException {
+
+        gitHub.getClient().credentialProvider = ImmutableCredentialProvider.fromJwtToken("myToken");
+
+        OrgInstallationCredentialProvider provider = new OrgInstallationCredentialProvider("testOrganization", gitHub);
+
+        provider.getEncodedAuthorization();
+    }
+
+    @Test
+    public void validJWTTokenAllowsOauthTokenRequest() throws IOException {
+        gitHub.getClient().credentialProvider = ImmutableCredentialProvider.fromJwtToken("valid-token");
```

Comment:
Are the credentials returned from `fromJwtToken` actually immutable?  I thought the token expired every 10 minutes.